python/tvm/relay/qnn/ir_pass.py

```diff
@@ -0,0 +1,37 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#pylint: disable=unused-argument
```

Comment:
What I mean is that we should avoid using `#pylint: disable=unused-argument` on top of a Python script file. If an argument is really unused, follow the following convention to put the directive inside the specific function:

``` python
import say

def f(a):
    #pylint: disable=unused-argument
    return say.fmt("The value of 'a' is {a}")
```
This helps `pylint` check unused arguments in other functions inside the same Python script file.